{
    "checksum": "187970018a21d6e1fefecc3ee89e0c93ed72c73e62e48b26bb1ac7e6664d00ee",
    "file": "wrong-file-sha256-hash-in-tree.conf",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 6,
    "package": "backupchecker",
    "path": "backupchecker/1.7-3/functional-tests/wrong-file-sha256-hash-in-tree/wrong-file-sha256-hash-in-tree.conf",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/backupchecker/1.7-3/",
        "metric": {
            "size": 6452
        },
        "pts_link": "https://tracker.debian.org/pkg/backupchecker",
        "sloc": [
            [
                "python",
                8568
            ],
            [
                "makefile",
                161
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/backupchecker",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/backupchecker/1.7-3/functional-tests/wrong-file-sha256-hash-in-tree/wrong-file-sha256-hash-in-tree.conf",
    "stat": {
        "perms": "rw-r--r--",
        "size": 111,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.7-3"
}